corral/bundle/json.pony

```diff
@@ -21,7 +22,8 @@ primitive Json
     else
       (let err_line, let err_message) = json.parse_report()
       log.err(
-        "JSON error at: " + file.path.path + ":" + err_line.string() + " : " + err_message
+        "JSON error at: " + file.path.path + ":" + err_line.string() + " : " +
+          err_message
       )
```

Comment:
This parenthesis should immediately follow the last argument on the previous line.